### PR TITLE
Add documentation to deploy Sylius to SymfonyCloud

### PR DESCRIPTION
**Pull request by tucksaun**
| Q               | A
| --------------- | -----
| Branch?         | 1.6
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

Some customers asked us some basic configuration to deploy Sylius projects to SymfonyCloud, let 's make this available to everyone 😃

### CODE REVIEW

File: docs/cookbook/deployment/symfonycloud.rst [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+            mkdir -p public/media/image var/log
+            (>&2 symfony-deploy)
+
+4. Commit the configuration
+---------------------------
+
+.. code-block:: bash
+
+    $ git add php.ini .symfony .symfony.cloud.yaml && git commit -m "SymfonyCloud configuration"
```
Comment (lchrusciel) [inline docs/cookbook/deployment/symfonycloud.rst]: Did we have the rest of the project already committed? I mean, this is the first time `git` command appears, and I'm wondering if we shouldn't mention something about it earlier.

Comment (tucksaun) [reply docs/cookbook/deployment/symfonycloud.rst]: You are right I should probably add a simple additional step in section 1

Comment (tucksaun) [reply docs/cookbook/deployment/symfonycloud.rst]: added

**Comment by tucksaun** [comment]
thanks for review @CoderMaggie 👍 . I though about the index but not the map. this is updated

**Comment by lchrusciel** [comment]
Thanks, Tugdual! :1st_place_medal:

**Pull request merged**